packages/u/usbutils/package.yml

```diff
@@ -12,13 +12,15 @@ description: |
 builddeps  :
     - pkgconfig(libusb-1.0)
 setup      : |
-    %reconfigure --disable-zlib \
-        --datadir=/usr/share/hwdata \
-        --disable-usbids
+    # This patch adds usbreset binary to the package, since upstream does not consider it stable
+    # https://github.com/gregkh/usbutils/issues/222#issuecomment-2715192013
+    %patch -p1 -i $pkgfiles/usbreset.patch
+
+    %meson_configure --datadir=/usr/share/hwdata
 build      : |
-    %make
+    %ninja_build
 install    : |
-    %make_install
+    %ninja_install
 
     # Don't need this
     rm -rvf $installdir/%libdir%
```

Comment:
```suggestion
```

You can remove this altogether since that directory is not created with the new meson build